specification/reservations/resource-manager/Microsoft.Capacity/preview/2020-10-01/reservations.json

```diff
@@ -630,9 +630,284 @@
           }
         }
       }
+    },
+    "/providers/Microsoft.Capacity/operationResults/{operationResultId}": {
```

Comment:
we do not have to include this path in swagger because clients will take it directly from locations header